priv/session_lock_repo/migrations/20200918180620_guardiandb.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Ret.Repo.Migrations.CreateGuardianDBTokensTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:guardian_tokens, primary_key: false) do
+      add(:jti, :string, primary_key: true)
+      add(:aud, :string, primary_key: true)
+      add(:typ, :string)
+      add(:iss, :string)
+      add(:sub, :string)
+      add(:exp, :bigint)
+      add(:jwt, :text)
+      add(:claims, :map)
+      timestamps()
+    end
+  end
+end
```

Comment:
What are `session_lock_repo` migrations?

Reply:
The SessionLockRepo is a special repo that we created to work around issues with migrations on Hubs Cloud (See #365 if you really want to dive deeper). In short, as a side effect of having SessionLockRepo, `ecto.gen.migration` creates two migration files. The one in the `priv/session_lock_repo` directory is actually redundant and should be deleted. Ideally we can find a way to prevent `ecto.gen.migration` from creating these files in the first place.